process-migration-service/src/test/java/org/kie/processmigration/utils/ContextUtils.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.processmigration.utils;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.Properties;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Produces;
+
+import org.wildfly.swarm.container.config.ConfigViewFactory;
+import org.wildfly.swarm.container.config.ConfigViewImpl;
+import org.wildfly.swarm.spi.api.config.ConfigView;
+
+@ApplicationScoped
+public class ContextUtils {
+
+    @Produces
+    private ConfigView createConfigView() throws IOException {
+        URL url = getClass().getClassLoader().getResource("h2-defaults.yml");
```

Comment:
I guess `h2-defaults.yml` is available inside thorntail by default?

Reply:
The whole class is not needed anymore. Thanks for noticing